src/components/EntityInfo.vue

```diff
@@ -180,6 +195,29 @@ export default class EntityInfo extends Mixins(EnumMixin) {
     // assume Business Profile URL is always reachable
     window.location.assign(businessProfileUrl)
   }
+
+  /** Track route for breadcrumb. */
+  @Watch('$route', { immediate: true })
+  private getBreadCrumbs (): Array<BreadcrumbInterface> {
```

Comment:
Interesting. So this is both an event callback and a getter?

Wouldn't a getter suffice?

Reply:
Not sure, will have to look into this!

Reply:
A getter will suffice 👍